src/components/Post/Createpost.js

```diff
@@ -36,6 +65,45 @@ export default class CreatePost extends React.Component {
         {/* <div className="headerContent">
           {`Post to ${channel.name}`}
         </div> */}
+        <div className="postAddons">
+          <div style={{cursor: 'pointer'}}>
+            <MaterialIcon
+              icon="image"
+              color='#000000'
+              size={30}
+              onClick={this.showUploadInterface}
+            />
+          </div>
+          {this.state.uploadInterface &&
+            <div>
+              <ImageUploader
+                withPreview
+                withIcon={false}
+                withLabel={false}
+                singleImage={true}
+                buttonText='Choose an image'
+                buttonStyles={{
+                  display: this.state.image ? 'none' : 'inline',
+                  height: '30px',
```

Comment:
This seems like it belongs in a stylesheet

Reply:
Can you do conditional styling in a style sheet? (I genuinely do not know)

Reply:
Tried to convert it to a stylesheet, remembered why I did it this way. The library I'm using for photo upload is stupid and overrides any style sheet styling with their own unless you provide them in this prop. It's a down-the-road todo to reverse engineer this library with our own code.

Reply:
I would put all the static stuff in the stylesheet, and then append the dynamic stuff in the js file. You can leave it as is if you want though, not a huge deal either way